src/nvim/os/input.c

```diff
@@ -61,10 +61,11 @@ void input_start_stdin(int fd)
 void input_stop_stdin(void)
 {
   if (!read_stream) {
-    return;
+    input_start_stdin(0);
```

Comment:
> Does libuv change the flags, even if we don't use stdin?

@splinterofchaos As far as I can tell, yes. Or we're `uv_init`ing it indirectly in some other way.

So I'm not really happy with this PR, I will probably end up creating a new function like `os_set_fd_blocking()` (?).